packages/fxa-auth-server/config/dev.json

```diff
@@ -35,4 +35,47 @@
     "key": "Correct_Horse_Battery_Staple_1",
     "maxTTL": "28 days"
   },
+  "subhub": {
+    "useStubs": true,
+    "stubs": {
+      "plans": [
+        {
+          "plan_id": "123doneProMonthly",
+          "product_id": "123doneProProduct",
+          "interval": "month",
+          "amount": 50,
+          "currency": "usd"
+        },
+        {
+          "plan_id": "321doneProMonthly",
+          "product_id": "321doneProProduct",
+          "interval": "month",
+          "amount": 50,
+          "currency": "usd"
+        },
+        {
+          "plan_id": "allDoneProMonthly",
+          "product_id": "allDoneProProduct",
+          "interval": "month",
+          "amount": 50,
+          "currency": "usd"
+        }
+      ]
+    }
+  },
+  "subscriptions": {
+    "managementClientId": "3c49430b43dfba77",
```

Comment:
Do we need to create a new OAuth client specifically for issuing the tokens for the off-domain subscriptions management pages?

Reply:
This is where I kind of cheated to get tests passing - this ID is the "Android Components Reference Browser" client used in `test/remote/oauth_tests.js`. Probably needs to be a completely new client

Reply:
I think I'm going to file a follow-up issue to address this in the content server work